build.xml

```
@@ -35,8 +35,8 @@
         <phingcall target="composer-check" />
     </target>
 
-    <target name="standards" depends="phplint,ecs,phpstan-packages,twig-lint,yaml-lint,eslint-check,standards-packages,standards-utils" description="Checks coding standards in the project-base, packages, microservices and utils folder."/>
-    <target name="standards-diff" depends="phplint-diff,ecs-diff,phpstan-packages,twig-lint-diff,yaml-lint,eslint-check-diff,standards-packages, standards-utils" description="Checks coding standards on changed files in the project-base, packages, microservices and utils folder."/>
+    <target name="standards" depends="phplint,ecs,phpstan-packages,twig-lint,yaml-lint,eslint-check,standards-packages,standards-utils,check-correct-yaml-indents" description="Checks coding standards in the project-base, packages, microservices and utils folder."/>
+    <target name="standards-diff" depends="phplint-diff,ecs-diff,phpstan-packages,twig-lint-diff,yaml-lint,eslint-check-diff,standards-packages, standards-utils,check-correct-yaml-indents" description="Checks coding standards on changed files in the project-base, packages, microservices and utils folder."/>
```

```suggestion
    <target name="standards-diff" depends="phplint-diff,ecs-diff,phpstan-packages,twig-lint-diff,yaml-lint,eslint-check-diff,standards-packages, standards-utils,yaml-standards" description="Checks coding standards on changed files in the project-base, packages, microservices and utils folder."/>
```
but i am not sure whether there is diff functionality